spec/order_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'order'
+
+describe Order do
+
+  let(:order_double) { double :order }
```

Comment:
This is where naming things the same gets confusing. It's unclear if this is doubling an `Order` object or an `@order` instance variable belonging to such an object. I presume it's the latter because we want to be testing real `Order`s, but you never know.